scripts/send-submissions-to-prepare-for-export/run.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+if [ "$#" -ne 1 ]; then
+  echo "The number of parameters is not correct. You need to provide the following: ./run.sh ipContainerPrepareSubmissionsForExport"
+  exit 1
+fi
+
+CONTAINER_IP=$1
+
+while IFS="" read -r l || [ -n "$l" ]
+do
+  curl --location --request POST $CONTAINER_IP/delta \
+  --header 'Content-Type: application/vnd.api+json' \
+  --data-raw ' [
+    {
+      "deletes": [],
+      "inserts": [
+        {
+          "subject": { "type": "uri", "value": "'$l'" },
```

Comment:
This was more a suggestion from the VSCode shellcheck extension. It refers to [this potential issue](https://www.shellcheck.net/wiki/SC2086) that may occur if double-quotes are not wrapped around variables.

```suggestion
          "subject": { "type": "uri", "value": "'"$l"'" },
```